openmetadata-ui/src/main/resources/ui/src/components/Tag/TagsContainerV2/TagsContainerV2.tsx

```diff
@@ -152,7 +156,11 @@ const TagsContainerV2 = ({
     () =>
       showAddTagButton ? (
         <Col className="m-t-xss" onClick={handleAddClick}>
-          <TagsV1 startWith={TAG_START_WITH.PLUS} tag={TAG_CONSTANT} />
+          <TagsV1
+            startWith={TAG_START_WITH.PLUS}
+            tag={isGlossaryType ? GLOSSARY_CONSTANT : TAG_CONSTANT}
+            tagType={tagType}
+          />
```

Comment:
same here, can addition of new prop be avoided here?

Reply:
Explained the need above